internal/config/network.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020-2023 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package config
+
+import (
+	"github.com/decred/dcrd/chaincfg/v3"
+)
+
+type Network struct {
+	*chaincfg.Params
+	DcrdRPCServerPort   string
+	WalletRPCServerPort string
+	BlockExplorerURL    string
+	// MinWallets is the minimum number of voting wallets required for a vspd
+	// deployment on this network. vspd will log an error and refuse to start if
+	// fewer wallets are configured.
+	MinWallets int
+	// DCP0005Height is the activation height of DCP-0005 block header
+	// commitments agenda on this network.
+	DCP0005Height int64
+}
+
+var MainNet = &Network{
```

Comment:
Probably not a big deal in vspd, but making these pointers means they have to be scanned on every single GC pass for the entire life of the process which eats extra CPU time.  They should be concrete structs for best performance and then only have their addresses taken at the call sites.

Reply:
Ah interesting! I noticed you making that change to `rpcclient.ConnConfig` in the `newHub` func in the PR decred/dcrpool#356. I wondered what difference it would make, now I know!

I'll update that.